fixtures/component_fixtures/EC2/component.yaml

```diff
@@ -12,7 +12,11 @@ satisfies:
     system_key: CloudFoundry
     verification_key: UAA_Verification_1
   implementation_status: partial
-  narrative: Justification in narrative form
+  narrative:
+    - key: "a"
+      text: "Justification in narrative form A for CM-2"
+    - key: "b"
+      text: "Justification in narrative form B for CM-2"
```

Comment:
we will. which also made me think we should have a `lib` folder in that repository. 
for example:
```
schema/
-- lib/
---- go/
------v1_0/
--------component.go
------v2_0/
--------component.go
------v3_0/
--------component.go
```
Then we would ensure that the schema repository stays up to date because you would have to import instead of using this internal component.go

Thoughts?